components/application-api/kustomization.yaml

```diff
@@ -0,0 +1,10 @@
+resources:
+- allow-argocd-to-manage.yaml
+- argocd-permissions.yaml
+- https://github.com/redhat-appstudio/application-api/config/crd?ref=77cba9006505d430248f863f81539cd8b39efaed
+
+
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: application-api
```

Comment:
just out of curiosity - is there anything namespace-scoped?

Reply:
I added it to be consistent, but since there's nothing namespace scoped it can be removed

Reply:
it was a question just out of curiosity - you didn't have to remove it :wink: :smile: 
but either way is fine :+1: